G1ANT.Addon.Selenium/G1ANT.Addon.Selenium/Commands/SeleniumRunScriptCommand.md

```diff
@@ -1,36 +1,37 @@
+| `errorcall`    | [procedure](G1ANT.Language/G1ANT.Language/Structures/ProcedureStructure.md) | no       |                                                             | Name of a procedure to call when the command throws an exception or when a given `timeout` expires |
+| `errorjump`    | [label](G1ANT.Language/G1ANT.Language/Structures/LabelStructure.md) | no       |                                                             | Name of the label to jump to when the command throws an exception or when a given `timeout` expires |
+| `errormessage` | [text](G1ANT.Language/G1ANT.Language/Structures/TextStructure.md) | no       |                                                             | A message that will be shown in case the command throws an exception or when a given `timeout` expires, and no `errorjump` argument is specified |
+| `errorresult`  | [variable](G1ANT.Language/G1ANT.Language/Structures/VariableStructure.md) | no       |                                                             | Name of a variable that will store the returned exception. The variable will be of [error](G1ANT.Language/G1ANT.Language/Structures/ErrorStructure.md) structure  |
 
-For more information about `if`, `timeout`, `errorjump` and `errormessage` arguments, please visit [Common Arguments](https://github.com/G1ANT-Robot/G1ANT.Manual/blob/master/G1ANT-Language/Common-Arguments.md)  manual page.
+For more information about `if`, `timeout`, `errorcall`, `errorjump`, `errormessage` and `errorresult` arguments, see [Common Arguments](G1ANT.Manual/appendices/common-arguments.md) page.
 
-This command is contained in **G1ANT.Addon.Selenium.dll**.
-See: [https://github.com/G1ANT-Robot/G1ANT.Addon.Selenium](https://github.com/G1ANT-Robot/G1ANT.Addon.Selenium)
+## Example
 
-**Example 1:**
-
-In this example, G1ANT.Robot opens 'google.com' and uses Javascript to return **.tagName** of some component on a web page. In our case the component is 'body'. And to select this 'body' element we need to use Javascript structure- **document.querySelector("body")**. It is a standard Javascript syntax used while choosing elements on a web page.
-Please, bear in mind it is crucial to use **return** word before Javascript syntax.
+In the following script the robot opens Google site and uses Javascript to return the `.tagName` of some component on the webpage — `body`, in this case. To select this `body` element a Javascript structure is used: `document.querySelector("body")`. It is a standard Javascript syntax for choosing elements on a webpage:
 
 ```G1ANT
-selenium.open type ‴firefox‴ url ‴google.com‴
-selenium.runscript script ‴return document.querySelector("body").tagName‴ result ♥result2
-dialog ♥result2
+selenium.open chrome url google.com
+selenium.runscript ‴return document.querySelector("body").tagName‴
+dialog ♥result
 ```
+
+> **Note:** It is crucial to use `return` keyword before Javascript code.
```

Comment:
Nope. **If you want your Javascript code to return something** then it is crucial to use this keyword. If you want it to just do something (click somewhere for example) then you don't need to add "return" word before your Javascript code using this command :)

Reply:
```suggestion
> **Note:** It is crucial to use `return` keyword before Javascript code if this code is designed to return something.
```